lib/apiservers/service/restapi/handlers/common.go

```diff
@@ -33,26 +33,37 @@ import (
 	"github.com/vmware/vic/pkg/vsphere/vm"
 )
 
-func buildData(ctx context.Context, url url.URL, user string, pass string, thumbprint *string, datacenter *string, computeResource *string) (*data.Data, error) {
+type BuildDataParams struct {
```

Comment:
I'd only expect this to be used within `handlers`. Does it need to be exported?

Reply:
Good catch, fixed